### PR TITLE
[plugin.video.nos] 4.0.2

### PR DESCRIPTION
**Pull request by artofcode**
### Description

Updated parser to new NOS website structure
### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

**Comment by Lunatixz** [comment]
Thanks for the update, can you correct the title of your submission and squash the commit into a single PR. Details in the document below:
https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md

**Comment by Rechi** [comment]
@artofcode you aren't the maintainer of this add-on. We usually only allow the maintainer to update them. Have you tried to contact him?

**Comment by artofcode** [comment]
> @artofcode you aren't the maintainer of this add-on. We usually only allow the maintainer to update them. Have you tried to contact him?

I have. I initially submitted a pull request on his own repository, but it was left ignored. I emailed him, but no reply unfortunately. As the plugin isn't working at all without these changes, I decided to go ahead and try it through this route instead.

File: plugin.video.nos/addon.xml [hunk 1]
```diff
@@ -11,7 +11,9 @@
     <provides>video</provides>
   </extension>
   <extension point="xbmc.addon.metadata">
-    <news>v4.0.1 (2018-08-11)
+    <news>v4.0.2 (2019-07-25)
```
Comment (enen92) [inline plugin.video.nos/addon.xml]: This is only for the last version

**Comment by enen92** [comment]
Since the PR in the original maintainer repository was not commented and there aren't any signs of activity, I recommend to do the following:

- Add yourself as an author, after the original author (keep the credits)
- Change the upstream source code location to your own repo/branch
- Provide credits somewhere on the readme
- Squash the commits

**Comment by artofcode** [comment]
Hi @enen92, I've made the changes and squashed the commits, but I screwed something up in the process, resulting in ee87ca0. Could you advise me how I can fix this?

**Comment by enen92** [comment]
I'd say:

```git log```

To find the commit just before your first commit. Then:

```
git rebase -i last_commit_before_yours
```
In the editor keep the first line as `pick` and change the others to `squash`. Save the file and change the commit message to "[plugin.video.nos] version". Finally:
```
git push -u origin gotham --force
```

**Comment by artofcode** [comment]
Strangely enough, that particular entry [ee87ca0](https://github.com/xbmc/repo-plugins/pull/2461/commits/ee87ca060d73f766b1924ad24065f636fa02a0c5) doesn't show up in git log.

Also when trying to do a rebase, I don't get it as an option either.

**Comment by Rechi** [comment]
To fix your pull request execute the following commands (origin is your repo, upstream is the xbmc repo)

```
git fetch upstream
git checkout gotham
git reset --hard upstream/gotham
git checkout origin/gotham -- plugin.video.nos/
git commit -m '[[plugin.video.nos] 4.0.2'
git push -f origin gotham
```

File: plugin.video.nos/addon.xml (hunk 1 above)
Comment (enen92) [reply plugin.video.nos/addon.xml]: In the next version please use the news tag only for the changes in the current version

**Comment by Rechi** [comment]
@artofcode this add-on isn't Python 3 compatible. Future updates have to be compatible with Python 3.
Have a look at https://kodi.tv/article/attention-addon-developers-migration-python-3 for more information.

**Comment by artofcode** [comment]
@enen92 / @Rechi : thanks for your help with this!
I'll have a look at starting the migration of the plugin to python 3.

**Pull request merged**